.openhands/microagents/test-style.md

```
@@ -0,0 +1,124 @@
+## 基本方針
+
+このリポジトリのテストコードでは、`github.com/stretchr/testify` パッケージを使用して簡潔で読みやすいテストを記述します。
+
+## 使用するアサーション
+
+- `assert`: 一般的なアサーションに使用（テスト継続）
+- `require`: 重要な条件のアサーションに使用（失敗時テスト中断）
+
+## テストコード作成ルール
+
+### 基本構造
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)